cli/azd/cmd/cobra_builder.go

```diff
@@ -91,14 +122,33 @@ func (cb *CobraBuilder) configureActionResolver(cmd *cobra.Command, descriptor *
 
 	// Only bind command to action if an action resolver had been defined
 	// and when a RunE hasn't already been set
-	if descriptor.Options.ActionResolver == nil || cmd.RunE != nil {
+	if cmd.RunE != nil {
+		return nil
+	}
+
+	// Neither cmd.RunE or descriptor.Options.ActionResolver set.
+	// Set default behavior to either --docs or help
+	if descriptor.Options.ActionResolver == nil {
```

Comment:
correct.
This is also applies to `azd` alone.  
We don't provide neither an actionResolver or a run/runE for those commands.
By default, cobra invokes the `help` commmand for these cases and consider those commands as `non-runnable`.

In this PR, we are re-defining what it means to be a `non-runnable` at azd-cmd-level. Instead of directly invoking the command's help, we check for `--docs` or any other flag like this which we might want to have in the future.

As a side effect, yes, we are making all commands to be runnable commands to the eyes of Cobra.